### PR TITLE
doc: update git repo for CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by wangyao2016**
Since hugegraph was donated to Apache Foundation, the GitHub repo address has also changed.
So I just update CONTRIBUTING.md when I read it.

I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Review by javeme — APPROVED** [review]
Thanks for your contribution

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,22 +14,22 @@ Before submitting the code, we need to do some preparation:
...
    # clone code from remote to local repo
-   git clone https://github.com/${GITHUB_USER_NAME}/hugegraph
+   git clone https://github.com/${GITHUB_USER_NAME}/incubator-hugegraph.git
    ```
 
 4. Configure local HugeGraph repo
 
    ```shell
-   cd hugegraph
+   cd incubator-hugegraph
```
Comment (javeme) [inline CONTRIBUTING.md:29]: we can keep hugegraph name

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -14,22 +14,22 @@ Before submitting the code, we need to do some preparation:
...
-2. Fork HugeGraph repo from GitHub: [https://github.com/hugegraph/hugegraph/fork](https://github.com/hugegraph/hugegraph/fork)
+2. Fork HugeGraph repo from GitHub: [https://github.com/apache/incubator-hugegraph/fork](https://github.com/apache/incubator-hugegraph/fork)
 
-3. Clone code from fork repo to local: [https://github.com/${GITHUB_USER_NAME}/hugegraph](https://github.com/${GITHUB_USER_NAME}/hugegraph)
+3. Clone code from fork repo to local: [https://github.com/${GITHUB_USER_NAME}/incubator-hugegraph](https://github.com/${GITHUB_USER_NAME}/incubator-hugegraph)
 
    ```shell
    # clone code from remote to local repo
-   git clone https://github.com/${GITHUB_USER_NAME}/hugegraph
+   git clone https://github.com/${GITHUB_USER_NAME}/incubator-hugegraph.git
```
Comment (javeme) [inline CONTRIBUTING.md:23]: prefer to keep "hugegraph" directory name: `git clone git-url directory-name`

**Review by javeme — CHANGES_REQUESTED** [review]

**Review by javeme — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (wangyao2016) [reply CONTRIBUTING.md]: Done.
THANKS

File: CONTRIBUTING.md (hunk 1 above)
Comment (wangyao2016) [reply CONTRIBUTING.md]: Yeah, done.
THANKS

**Review by imbajin — APPROVED** [review]

**Pull request merged**